app/screens/send-bitcoin-screen/payment-details/onchain.ts

```diff
@@ -22,6 +22,8 @@ export const createNoAmountOnchainPaymentDetails = <T extends WalletCurrency>(
   params: CreateNoAmountOnchainPaymentDetailsParams<T>,
 ): PaymentDetail<T> => {
   const {
+    // FIXME: is that coming from usePriceConversion()
+    // in that case we should not pass it as a param
```

Comment:
Whats your thinking behind this?

Reply:
Simplifying the Params passed via route. It makes mocking/testing screen easier/ less boilerplat-y. 

But the question here is: is it the same function as the one returned by the hook or do you wrap it somehow? Didn't check into details?

Reply:
It is the same function from the hook, but it is important that the function is part of the object as it is used for the conversion to the settlement amount and it ensures that we can freeze the price at a particular point which is important when the user has confirmed a settlment amount and validation has been performed on that settlement amount.

Reply:
ok got it.